src/test/java/com/stripe/StripeMockProcess.java

```diff
@@ -0,0 +1,153 @@
+package com.stripe;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.lang.reflect.Field;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import lombok.Getter;
+
+public class StripeMockProcess {
+
+  private static final String ROOT = "src/test";
+  private static final String FIXTURE_PATH = "/openapi/fixtures3.json";
+  private static final String SPEC_PATH = "/openapi/spec3.json";
+
+  private static Process process;
+  @Getter
+  private static String port;
+  private static String pid;
+
+  /**
+   * Start embedded stripe-mock when spec and fixture resources are detected.
+   *
+   * @return whether the stripe-mock can be started and is now running.
+   */
+  public static boolean start() throws IOException, InterruptedException, NoSuchFieldException,
+      IllegalAccessException {
+
+    if (!new File(getPathSpec()).exists() || !new File(getPathFixture()).exists()) {
+      return false;
+    }
+
+    if (process != null && isAlive(process)) {
+      System.out.println(String.format("stripe-mock is already running, port = %s", port));
+      return true;
+    } else {
+      // to reinitialize all properties
+      process = null;
+      port = null;
+      pid = null;
+    }
+
+    ProcessBuilder processBuilder = new ProcessBuilder()
+        .command(
+            "stripe-mock",
+            "-http-port",
+            "0", // stripe-mock to choose an available port
+            "-spec",
+            getPathSpec(),
+            "-fixtures",
+            getPathFixture()
+        );
+
+    try {
+      process = processBuilder.start();
+    } catch (IOException e) {
+      System.out.println(String.format(
+          "Error while starting stripe-mock, port = %d, fixtures = %s and %s, "
+              + "error message = %s",
+          port, getPathSpec(), getPathFixture(), e.getMessage()
```

Comment:
Can you update this message to remove the port (since it will be `null` now that stripe-mock chooses the port itself)?

Reply:
whoops my bad. fixed!